apps/docs/content/components/CheckboxGroup/index.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: CheckboxGroup
+description: Use CheckboxGroup to group related checkbox inputs with supporting content
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+## Usage
+
+The `CheckboxGroup` family of components provide a way to group related [Checkbox](/components/Checkbox) components together with supporting content such as labels, captions, and validation messages. It creates a semantic fieldset structure that helps maintain proper form hierarchy and accessibility.
```

Comment:
As it's already undergone various iterations, could we match the content from this page please? https://primer.style/components/checkbox-group

Reply:
Happy to make the change, but I just want to check before I do it as writing these docs was a decent chunk of work this morning 😅

Is the request to ditch what I've currently got and copy/paste the page from the PRC docs, updating any minor things that differ between the two implementations?

Reply:
> Happy to make the change, but I just want to check before I do it as writing these docs was a decent chunk of work this morning 😅

Sorry @joshfarrant 😓 I should have mentioned this earlier. For future reference, you only add the React docs if when you're adding a new React component, the interface guidelines can land later unless it's easy enough to add in the first instance. Usually interface guidelines should include images too, but again can be added in a fast-follow PR if needed.

> Is the request to ditch what I've currently got and copy/paste the page from the PRC docs, updating any minor things that differ between the two implementations?

These two components are a bit unique in that the product counterpart already has mature documentation around this. Also given that our component shares the same API and usage guidelines as product and is essentially the same thing, my recommendation is to keep the content and structure of the guidelines as consistent as possible. You're fine to tweak aspects of the content to suit the unique aspects of our component though.

Reply:
No that's all good, thank you for the additional context.

I'll tweak the interface docs to match PRC shortly. Are you happy with the react docs as they are, or shall I do the same there?

Reply:
I've just replaced the interface guidelines with a (near) copy of PRC, lmk what you think

Reply:
There's only the default example in the product React docs, so I think we're good on that front 👍